taipy/core/reason/_reason_factory.py

```diff
@@ -22,3 +22,11 @@ def _build_data_node_is_not_written(dn_id: DataNodeId) -> str:
 
 def _build_not_submittable_entity_reason(entity_id: str) -> str:
     return f"Entity {entity_id} is not a submittable entity"
+
+
+def _build_config_can_not_create_reason(config_id: str) -> str:
+    return f'Object "{config_id}" is not a valid config to be created'
```

Comment:
```suggestion
    return f'Object "{config_id}" is not a valid config.'
```

Reply:
This is to vague, no?

Reply:
Yes. This message will be displayed in the UI to the end user.
What about:
```f"'{config_id}' must be a valid {entity_type} configuration."```
 That means we need to pass the entity_type as a parameter.
 It will be instantiated as: 
```
"'toto' must be a valid Scenario configuration."
```
or
```
"'toto' must be a valid DataNode configuration."
```

What do you think?

Reply:
let me try something. This is pretty hard since we have a generic `taipy.can_create()` API, which accept both DataNodeConfig and ScenarioConfig